velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -466,11 +473,20 @@ class ExpressionFuzzer {
 
   std::vector<core::TypedExprPtr> generateArgs(const CallableSignature& input) {
     std::vector<core::TypedExprPtr> inputExpressions;
-    inputExpressions.reserve(input.args.size());
+    auto variableArity = input.variableArity;
+    auto numVarArgs = !variableArity
+        ? 0
+        : folly::Random::rand32(FLAGS_max_num_varargs + 1, rng_);
+    inputExpressions.reserve(input.args.size() + numVarArgs);
```

Comment:
There will be at most `FLAGS_max_num_varargs + 1` arguments including the original argument in the function signature. Is this intended? If so, could you clarify in the comment for this flag?

Reply:
I think this should be self-explanatory. num_varargs means number of variadic arguments. For example func(A a, B b, ...) the number of variadic arguments would be the number of "..." we have as they are the variadics. I can add some comments for clarification though.

There will be at most FLAGS_max_num_varargs arguments instead of FLAGS_max_num_varargs + 1. The "+1" is for rand gen to gen [0, FLAGS_max_num_varargs] inclusive.

Reply:
In the code above we create conjuctSignature by `conjunctSignature.args = {BOOLEAN(), BOOLEAN()}`. So I thought we would do `inputExpressions.reserve(2 + numVarArgs)` here. Am I missing anything?